chapters/en/slides/module6/module6_02.Rmd

```diff
@@ -0,0 +1,352 @@
+
+The default projection method for maps is often `mercator`,
+but it causes heavy distortion of the relative areas of land masses,
+which get worse the closer to the poles we come.
+
+Fortunately,
+many people are moving away from using the projection
+because it gives an incorrect impression of how the world looks.
+A commonly used alternative is the `equalEarth` projection,
+which focuses on correctly representing the relative areas of all landmasses.
+
+Some of the most noticeable differences in this projection
+are that Africa appears much bigger
+and Greenland, Antarctica and Russia appear notably smaller.
+Africa is in fact 15x the area of Greenland,
+far from the `mercator` projection which shows them as roughly the same size.
+
+---
+
+## The fill and border colors of countries can be changed
+
+```{python}
+alt.Chart(world_map).mark_geoshape(
+    color='black', stroke='#706545', strokeWidth=1
+).project(type='equalEarth')
+```
+
+Notes:
+We can change the color of both the filled are and the border 
+for each country.
+Just like with other colors in Altair,
+we can use either a named color
+or and exact hex-code.
+
+Here we also make the stroke width slightly larger 
+so that it is easier to see
+the country borders.
+
+---
+
+## We can zoom and pan the map
+
+```{python}
+alt.Chart(world_map).mark_geoshape(
+    color='#2a1d0c', stroke='#706545', strokeWidth=0.5
+).project(type='equalEarth', scale=500, translate=[140, 610])
+```
+
+Notes:
+By default Altair automatically adjusts the projection
+so that all the data fits within the width and height of the chart.
+
+We can change this via the projections parameters `scale` (zoom level) and `translate` (panning).
+Here we adjust them to zoom in and center the map on Europe.
+Getting the panning right when zooming in can take a bit of fiddling.
+
+---
+
+## Working with individual countries can show regional details
+
+```{python}
+state_map = alt.topo_feature(data.us_10m.url, 'states')
+alt.Chart(state_map).mark_geoshape().project(type='equalEarth')
+```
+
+Notes:
+If we want information on regions within a country,
+we have to use a geo/topojson file that contains this info.
+There is already one in the vega demo datasets for the US,
+which is the one we are using here.
+
+The loading of the data is exactly the same as previously,
+but as you can see,
+the `equalEarth` projection is not very suitable for a single country.
+
+The US it is very small and both Alaska and Hawaii 
+are shown far away from the rest of the country.
+
+---
+
+## There are special projections for some countries such as the US
+
+```{python}
+alt.Chart(state_map).mark_geoshape().project(type='albersUsa')
+```
+
+Notes:
+There special US projection `albersUsa`
+is much preferred when only showing this country.
+You might recognize the projection from maps that you have seen elsewhere
+such as on the news.
+Characteristic is the placement of Alaska and Hawaii
+close to the rest of the country.
```

Comment:
```suggestion
To fix this, we can use a special projection `albersUsa`
that was created to visualize maps of the United States. 
It is a much preferred projection when only showing this country.

You might recognize the projection from maps that you have seen 
outside of this course, such as on the news. Of note, is the characteristic
placement of Alaska and Hawaii close to the rest of the country.
```

Reply:
What about other countries? What projections should we use for them? Given that our course is based in Canada, we should at least discuss that country.

Reply:
It was actually pretty complicated to get Canada with provinces working when we tried with a group in 532, I'll see if I can solve it but marking as nice to have for now (it might be just be a matter of finding the right format of the data online). Ideally we would read a local geojson also to show that... time....